### PR TITLE
Issue #99: Add table of contents for user guide

### LINKED ISSUE #99: Add Table of Contents for User Guide

### PR DESCRIPTION
**Pull request by domlimm**
Closes #99.

### CODE REVIEW

**Review by lzan98 — APPROVED** [review]
LGTM! I just have a dumb little question 😆

**Review by LeeEnEn — APPROVED** [review]
LGTM!

File: docs/UserGuide.md [hunk 1]
```diff
@@ -8,8 +8,8 @@ the process of hiring undergraduate/graduate Teaching Assistants (TA). They will
 data easily and review qualifications or availability for scheduling interviews. It is **optimized for use via a
 Command Line Interface (CLI)** while still having the benefits of a Graphical User Interface (GUI). If you can type fast, this application will be able to help you manage all things under the hood of the TA initiative faster than traditional GUI applications.
 
-
 * Table of Contents
+{:toc}
```
[the remark this answers is not in the document]
Comment (domlimm) [reply docs/UserGuide.md:12]: Hey @lzan98! Not a dumb question leon! A question is never ever dumb!!! It's an auto generator by markdown for Table of Contents which absorbs the headers i.e. `#`, `##`, `###`, etc.

**Comment by domlimm** [comment]
@lzan98 @LeeEnEn thank you both for reviewing! Can someone please merge in. Thank you! ❤️

**Pull request merged**